packages/ui/src/Typography/Typography.tsx

```diff
@@ -1,21 +1,41 @@
 import React from "react";
-import {
-    Typography as RwmcTypography,
-    TypographyProps as RmwcTypographyProps
-} from "@rmwc/typography";
+import { TypographyProps as RmwcTypographyProps } from "@rmwc/typography";
+
+import { Heading, HeadingLevels, Text } from "@webiny/ui-new";
 
 interface TypographyProps extends RmwcTypographyProps {
     children?: React.ReactNode;
     className?: string;
+    /**
+     * @deprecated
+     */
     style?: React.CSSProperties;
     tag?: string;
 }
 
 /**
- * Use Ripple component to display a list of choices, once the handler is triggered.
+ * @deprecated This component is deprecated and will be removed in future releases.
```

Comment:
Question: does your IDE highlight the deprecation of the component when it's used?

Reply:
Yes 👍 

![CleanShot 2024-08-21 at 11 40 49](https://github.com/user-attachments/assets/363a5f4f-d783-4146-87bb-fab6126edf66)